### DISCUSSION TITLE
"indexerror: list index out of range" the script is working for me but i'm getting a lot of errors too

### DISCUSSION THREAD

**Discussion by jdnewland** [itsToggle/plex_debrid#84] [Q&A] [answered]
I'm just wondering what this could be if it's something on my end or something in the script? the first version of the script did not do this but this new version is doing it.
![error](https://user-images.githubusercontent.com/5415055/185797877-468af4d9-28ca-4723-b267-6be603091439.png)

**Comment by itsToggle** [comment, marked as answer]
This should be fixed in the latest version from a few hours ago :)
